modules/internal/OwnedObject.chpl

```diff
@@ -373,7 +373,7 @@ module OwnedObject {
     ``lhs``, if any. Transfers ownership of the object managed by ``rhs``
     to ``lhs``, leaving ``rhs`` storing `nil`.
   */
-  proc =(ref lhs:_owned,
+  operator =(ref lhs:_owned,
```

Comment:
I'd think these owned/shared cases should be operator methods (but it wouldn't surprise me if that will take a compile bug fix to get working).